.dockerignore

```diff
@@ -3,4 +3,6 @@
 # except what we need
 !/bin/
 !/lib/
+!/sample/
```

Comment:
sample files will be used to make a initial test of the run, to have a small sql file instead of a very large one